apps/tangle-dapp/utils/calculateBnPercentage.ts

```diff
@@ -33,7 +33,7 @@ function calculateBnPercentage(amount: BN, total: BN): number {
   // Round the percentage to 2 decimal places. It's suitable to use
   // 2 decimal places since the purpose of this function is to provide
   // a visual representation of the percentage in the UI.
-  return Math.round(percentage * 100) / 100;
+  return (percentage * 100) / 100;
```

Comment:
The `.round()` function is utilized here to ensure the percentage always remains at 2 decimal places. Is there any particular reason you wish to remove it?